scripts/linters/pre_commit_linter.py

```diff
@@ -532,11 +578,11 @@ def _get_task_output(lint_messages, failed, task):
     return failed
 
 
-def _print_errors_stacktrace(errors_stacktrace):
+def _print_errors_stacktrace(errors_stacktrace: List[Optional[str]]) -> None:
```

Comment:
It can be `None` because the type of `stack_trace` is as follows

https://github.com/oppia/oppia/blob/58d1033f97b1270e4c95133db7d52b94f4913fb4/scripts/concurrent_task_utils.py#L106

It is appended to the list over here

https://github.com/oppia/oppia/blob/58d1033f97b1270e4c95133db7d52b94f4913fb4/scripts/concurrent_task_utils.py#L162

should I append to the list only if the stacktrace is present?

Reply:
Hmm, I would say that if we have `task.exception` but no `task.stacktrace` then we probably have some kind of issue and we should print something else instead of the stacktrace.

Reply:
Yeah that makes sense. 

done